src/core/markdown.js

```diff
@@ -88,7 +88,8 @@ const inlineElems = new Set([
 
 class Renderer extends marked.Renderer {
   code(code, language, isEscaped) {
-    if (language === "webidl") {
+    // regex to check whether the language is webidl
+    if (/(^webidl$|^idl$)/i.test(language)) {
       return `<pre class="idl">${code}</pre>`;
```

Comment:
regarding @saschanaz's point about `idl` and `webidl` -
to confirm - it is okay that we're currently tagging `webidl` stuff as `<pre class="idl">`, right?

cc @marcoscaceres

Reply:
Yes, for backward compatibility. Should we also `pre.webidl`? @marcoscaceres

Reply:
> Should we also allow `pre.webidl`?

Yes, that would be fine. Should we do that as a followup? that would require changing core/webidl.

Reply:
@janiceshiu, could you please file a follow-up bug to support the above?

Reply:
[filed!](https://github.com/w3c/respec/issues/2614)